### DISCUSSION TITLE
[双拼输入方案讨论]可否在双拼基础上，加入音调顶功。用a e o ;代表1 2 3 4声调

### DISCUSSION THREAD

**Discussion by gaboolic** [rime/home#1508] [Ideas]
例如
激素	jisu
急速	jisu e 
极速	jisu e;
寄宿	jisu ;;

自然码双拼、小鹤双拼以及拼音加加双拼的a e o ;正好可以用来作为音调顶功。
搜狗和微软不行，用了分号
智能abc好像也不行，只剩下u i ;可以用，o是零声母

**Comment by gaboolic** [comment]
4码起顶。
输入jisu后，再输入 非aeo;编码，直接上屏 激素
输入jisue后，候选框是急速 极速，再输入 非aeo;编码，直接上屏 急速

**Comment by mirtlecn** [comment]
有五个音调吧，还有轻声。

而且拼音重码率太高的，除了词典要手动解决大量同样音节、不同声调的词汇外，这么处理过后，顶功符合预期吗？

yi4 yi4 -> 意义 异议 意译 奕奕

![image](https://github.com/rime/home/assets/76689045/f31192ae-3288-44e0-90d9-d635b9b50c21)

**Comment by gaboolic** [comment]
轻声合并到一声了吧
自带的地球拼音有音调，没轻声
&nbsp;yiyi 44只留下最多三个词组就行，奕奕就算了
顶功顶意义



---Original---
From: ***@***.***&gt;
Sent at: 2024年3月31日(Sun) Evening11:11
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [rime/home] [双拼输入方案讨论]可否在双拼基础上，加入音调顶功。用a e o ;代表1 2 3 4声调 (Discussion #1508)




 
有五个音调吧，还有轻声。
 
而且拼音重码率太高的，除了词典要手动解决大量同样音节、不同声调的词汇外，这么处理过后，顶功符合预期吗？
 
yi4 yi4 -&gt; 意义 异议 意译 奕奕
 
image.png (view on web)
